### PR TITLE
Hh/first feature flag quickstart

### PR DESCRIPTION
**Pull request by hhargreaveseppo**
Creating a feature flag specific quickstart separate from the experiment flag quickstart.

### CODE REVIEW

File: docs/quick-starts/feature-flag-quickstart.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 slug: /feature-flag-quickstart
-sidebar_position: 3
+sidebar_position: 2
 ---
 
-# Running your first experiment
+# Creating your first feature flag
 
-This 10 minute guide will walk through running your first experiment with Eppo. In the example we'll imagine that we are testing a new checkout page.
+This 10 minute guide will walk through running your first feature flag with Eppo. In the example we'll imagine that we are testing a new checkout page.
```
Comment (lukasgoetzweiss) [inline docs/quick-starts/feature-flag-quickstart.md:8]:
```suggestion
This 10 minute guide will walk through creating your first feature flag in Eppo. In the example, we'll imagine that we are adding a basic on/off switch for a new onboarding page.
```

File: docs/quick-starts/feature-flag-quickstart.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 While Eppo feature flags can be used for feature gates, kill switches, and targeted rollouts, this guide will focus on using Eppo flags for running randomized experiments.
```
Comment (lukasgoetzweiss) [inline docs/quick-starts/feature-flag-quickstart.md:10]:
```suggestion
While Eppo's SDK can be used for targeted rollouts, A/B/n experiments, and personalization via Contextual Bandits, this guide will focus on creating a simple on/off switch in Eppo.
```

File: docs/quick-starts/feature-flag-quickstart.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
```
Comment (lukasgoetzweiss) [inline docs/quick-starts/feature-flag-quickstart.md:11]: I guess you can't propose changes on lines that are not part of the diff? 

We should drop this sentence for the feature flagging page:

> Note that if you are using Eppo alongside an existing randomization tool, you can skip right to the [next quick start](/experiment-quickstart).

File: docs/quick-starts/feature-flag-quickstart.md [hunk 4]
```diff
@@ -33,19 +33,19 @@ Give the flag a descriptive human readable name and create variations for each v
...
 
-### 3. Create an experiment allocation
+### 3. Create a Feature Gate allocation
 
 After creating the flag, switch into the Test environment:
 
 ![Feature gate 2](/img/feature-flagging/feature-flag-qs-2.png)
 
-Now that you're in the Test environment, add an Experiment allocation to your flag. If you want to force certain segments or users into one variant, you can also add a Feature Gate allocation. You can read more about using Eppo for Feature Gates [here](/feature-flagging/feature-gates).
+Now that you're in the Test environment, add a Feature Gate allocation to your flag. If you want to force certain segments or users into one variant, you can also add a Feature Gate allocation. You can read more about using Eppo for Feature Gates [here](/feature-flagging/feature-gates).
```
Comment (lukasgoetzweiss) [inline docs/quick-starts/feature-flag-quickstart.md:42]:
```suggestion
Now that you're in the Test environment, add a Feature Gate allocation to your flag:
```

File: docs/quick-starts/feature-flag-quickstart.md [hunk 5]
```diff
@@ -33,19 +33,19 @@ Give the flag a descriptive human readable name and create variations for each v
...
 
-![Feature gate 3](/img/feature-flagging/feature-flag-qs-3.png)
+![Feature gate 3](/img/feature-flagging/feature-gate-setup-1.png)
 
 For this example, we will assign all users to the experiment. If you want to target specific users, you can add targeting rules to the allocation. You can read more about targeting [here](/feature-flagging/targeting).
 
-![Feature gate 4](/img/feature-flagging/feature-flag-qs-4.png)
+![Feature gate 4](/img/feature-flagging/feature-gate-setup-3.png)
 
 ### 4. Connect a logging function to the Eppo SDK
```
Comment (lukasgoetzweiss) [inline docs/quick-starts/feature-flag-quickstart.md:50]: We can skip logging for the sake of this guide.

Comment (lukasgoetzweiss) [reply docs/quick-starts/feature-flag-quickstart.md:50]: If so, we should update the initialization code to not have an event logger

File: docs/quick-starts/feature-flag-quickstart.md [hunk 6]
```diff
@@ -113,11 +113,11 @@ return variation ? <NewCheckoutPage /> : <OldCheckoutPage />
 
 Note that the `get<Type>Assignment` methods in Eppo are deterministic, meaning that they will always return the same variant for a given subject (e.g., user) throughout the experiment.
 
-### 7. Turn on the flag to start splitting traffic
+### 7. Turn on the flag for users
```
Comment (lukasgoetzweiss) [inline docs/quick-starts/feature-flag-quickstart.md:116]:
```suggestion
### 7. Turn on the flag
```

**Review by lukasgoetzweiss — CHANGES_REQUESTED** [review]
I think this should focus on a full on/off switch. If the reader wants to randomly serve some percent of traffic, the "running your first experiment" quick start should be more what they are looking for

File: docs/quick-starts/feature-flag-quickstart.md (hunk 3 above)
Comment (hhargreaveseppo) [reply docs/quick-starts/feature-flag-quickstart.md:11]: done

File: docs/quick-starts/feature-flag-quickstart.md (hunk 5 above)
Comment (hhargreaveseppo) [reply docs/quick-starts/feature-flag-quickstart.md:50]: done

File: docs/quick-starts/feature-flag-quickstart.md [hunk 7]
```diff
@@ -113,11 +113,11 @@ return variation ? <NewCheckoutPage /> : <OldCheckoutPage />
 
 Note that the `get<Type>Assignment` methods in Eppo are deterministic, meaning that they will always return the same variant for a given subject (e.g., user) throughout the experiment.
```
[the remark this answers is not in the document]
Comment (hhargreaveseppo) [reply docs/quick-starts/feature-flag-quickstart.md:114]: Let me update the language, I could see users wanting to use a string or json type flag for flagging use cases -- but that might be in a category of more 'advanced' use cases that aren't necessary for a quickstart

**Pull request merged**